1.download-data/README.md

```diff
@@ -0,0 +1,7 @@
+# Download data
+Make sure you have python installed.
+
+Open your terminal to this location. Type python downloadData.py --downloadLocation [The path where you want to store your data]
```

Comment:
```suggestion
```bash
# Navigate to this directory
cd 1.download-data

# Running this script will download all data in the `data` folder
python downloadData.py
```
```

Reply:
I like the ability for users to download this data to any location, but for our purposes in this analysis, we should be explicit about where to expect data. All downstream analysis modules will use this data.

Reply:
You're right. I changed the readme but left the function option there